rest/model/patch.go

```diff
@@ -203,12 +203,16 @@ func (apiPatch *APIPatch) buildBasePatch(p patch.Patch) {
 	apiPatch.PatchNumber = p.PatchNumber
 	apiPatch.Author = utility.ToStringPtr(p.Author)
 	apiPatch.Version = utility.ToStringPtr(p.Version)
-	apiPatch.Status = utility.ToStringPtr(p.Status)
 	apiPatch.Hidden = p.Hidden
 	apiPatch.CreateTime = ToTimePtr(p.CreateTime)
 	apiPatch.StartTime = ToTimePtr(p.StartTime)
 	apiPatch.FinishTime = ToTimePtr(p.FinishTime)
 	apiPatch.MergedFrom = utility.ToStringPtr(p.MergedFrom)
+	apiPatch.Status = utility.ToStringPtr(p.Status)
+	// Ensure we're returning a consistent successful status.
```

Comment:
I assume the answer is yes, but one day we're going to delete `evergreen.LegacyPatchSucceeded` from the codebase entirely, right?

Reply:
yes, in a year haha (https://jira.mongodb.org/browse/EVG-20032)